### DISCUSSION TITLE
Huggin Chat being down most of the time

### DISCUSSION THREAD

**Discussion by batlahiya** [Soulter/hugging-chat-api#129] [Q&A]
Huggin chat has been not responding most of the time recently, is there any way to get pass this issue

**Comment by Whitelisted1** [comment]
Is this for the UI as well? If you are still able to use the [HuggingChat UI](https://huggingface.co/chat/), this would be more appropriate as an issue. Make sure you include information such as python version and logs if you are creating an issue.
